src/components/Book.js

```diff
@@ -1,47 +1,30 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
-import { CircularProgressbar } from 'react-circular-progressbar';
+import { RemoveBook } from '../redux/books/books';
 import Button from './Button';
 
-const Book = ({
-  category, title, author, chapter, completed,
-}) => (
-  <div className="book-container">
-    <>
-      <p id="category">{category}</p>
-      <h2 id="title">{title}</h2>
-      <p id="author">{author}</p>
+const Book = ({ title, author }) => {
+  const dispatch = useDispatch();
 
-      <div className="btn-container">
-        <Button buttonType="button" text="Comments" id="btn-comment" />
-        <Button buttonType="button" text="Remove" id="btn-remove" />
-        <Button buttonType="button" text="Edit" id="btn-edit" />
-      </div>
-    </>
-    <div className="progress">
-      <CircularProgressbar value={completed} text={`${completed}%`} />
+  return (
+    <div className="book-container">
       <>
-        <h3>{completed}</h3>
-        <p>completed</p>
+        <h2 id="title">{title}</h2>
+        <p id="author">{author}</p>
+
+        <div className="btn-container">
+          <Button buttonType="button" text="Remove" id="btn-remove" onClick={(book) => dispatch(RemoveBook(book))} />
```

Comment:
- [ ] As requested by the previous code reviewer, which is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md) for this project, " `Add book and remove book should work in the same way after the refactor.` " please ensure the Add book and remove book features should work the same as before even after the refactoring to use the API has been done.